drivers/flash/Kconfig.mcux

```diff
@@ -50,6 +71,17 @@ config FLASH_MCUX_FLEXSPI_NOR_WRITE_BUFFER
 	  This prevents faults when the data to write would be located on the
 	  flash itself.
 
+config FLASH_MCUX_FLEXSPI_HYPERFLASH_WRITE_BUFFER
+	bool "MCUX FlexSPI HYPERFLASH write RAM buffer"
+	default y
+	depends on FLASH_MCUX_FLEXSPI_HYPERFLASH
+	help
+	  Copy the data to a RAM buffer before writing it to the flash.
+	  This prevents faults when the data to write would be located on the
+	  flash itself.
+
+endif # USE_DEDICATED_FLEXSPI_FLASH_DRIVER
+
```

Comment:
```suggestion
```

Related to the `USE_DEDICATED_FLEXSPI_FLASH_DRIVER` comment.

Reply:
Made the choice optional so USE_DEDICATED_FLEXSPI_FLASH_DRIVER is obsolete.